xpdacq/calib.py

```diff
@@ -170,7 +170,7 @@ def _collect_calib_img(exposure, dark_sub_bool, calibration_instance,
     sample = Sample(bto, calibration_dict)
     uid = RE_instance(sample, ScanPlan(bto, ct, exposure))
     light_header = xpd_configuration['db'][uid[-1]]  # last one must be light
-    dark_uid = light_header.start.get('sc_dk_field_uid')
+    dark_uid = light_header.start.get('dark_client_uid')
```

Comment:
I think this may cause problems on the line below. I think you need a `db(dark_server_uid=dark_uid)[0]` type thing on the next line. Also we may want the `[0]` to be configurable